lib/bibliotheca/language.rb

```diff
@@ -31,36 +33,46 @@ def self.find_by_name(name)
         Language.new(name: 'haskell',
                      extension: 'hs',
                      ace_mode: 'haskell',
-                     devicon: 'haskell'),
+                     devicon: 'haskell',
+                     url: 'http://mumuki-hspec-server.herokuapp.com'),
         Language.new(name: 'java',
                      extension: 'java',
                      ace_mode: 'java',
-                     devicon: 'java'),
+                     devicon: 'java',
+                     url: 'http://162.243.111.176:8003'),
         Language.new(name: 'wollok',
                      extension: 'wlk',
-                     ace_mode: 'wollok'),
+                     ace_mode: 'wollok',
+                     url: 'http://mumuki-wollok-server.herokuapp.com'),
         Language.new(name: 'c',
                      extension: 'c',
-                     ace_mode: 'c'),
+                     ace_mode: 'c',
+                     url: 'http://162.243.111.176:8001'),
         Language.new(name: 'prolog',
                      extension: 'pl',
                      ace_mode: 'prolog',
-                     devicon: 'prolog'),
+                     devicon: 'prolog',
+                     url: 'http://mumuki-plunit-server.herokuapp.com'),
         Language.new(name: 'ruby',
                      extension: 'rb',
                      ace_mode: 'ruby',
-                     ruby_rough: 'ruby_rough'),
+                     ruby_rough: 'ruby_rough',
+                     url: 'http://162.243.111.176:8000'),
         Language.new(name: 'gobstones',
                      extension: 'gbs',
                      test_extension: 'yml',
-                     ace_mode: 'gobstones'),
+                     ace_mode: 'gobstones',
+                     url: 'http://mumuki-gobstones-server.herokuapp.com'),
         Language.new(name: 'javascript',
                      extension: 'js',
                      ace_mode: 'javascript',
-                     devicon: 'javascript_badge'),
+                     devicon: 'javascript_badge',
+                     url: 'http://mumuki-mocha-server.herokuapp.com'),
         Language.new(name: 'text',
                      extension: 'txt',
                      test_extension: 'yml',
-                     devicon: 'code')]
+                     devicon: 'code',
+                     url: 'https://mumuki-text-server.herokuapp.com')]
+
```

Comment:
We need to extract another service that indexes the languages and treats those URLs dynamically. 

How could we name it? `dictionary`? `encyclopedia`?